pkg/agent/run.go

```diff
@@ -181,7 +181,7 @@ func validate() error {
 	}
 
 	if !strings.Contains(string(cgroups), "memory") {
-		msg := "ailed to find memory cgroup, you may need to add \"cgroup_memory=1 cgroup_enable=memory\" to your linux cmdline (/boot/cmdline.txt on a Raspberry Pi)"
+		msg := "Failed to find memory cgroup, you may need to add \"cgroup_enable=memory\" to your linux cmdline (/boot/cmdline.txt on a Raspberry Pi)"
```

Comment:
Note that 'F' or 'f' is prepended to this below so that it's capitalized properly for both log and error use.